src/components/calcite-block-section/calcite-block-section.tsx

```diff
@@ -119,12 +119,12 @@ export class CalciteBlockSection {
       );
 
     return (
-      <Host aria-expanded={open ? "true" : "false"}>
+      <section aria-expanded={open ? "true" : "false"}>
```

Comment:
nitpick: There is a non-zero cost to adding extra DOM elements. 
https://gtmetrix.com/reduce-the-number-of-dom-elements.html
Applications like the mapmaker will see a potential performance benefit from anywhere we can trim them.

Let's just be sure the gains outweigh the losses from something like this.

Reply:
Noted, although I'm not too worried about this at this point.